code/game/objects/items/melee/energy.dm

```diff
@@ -204,7 +204,7 @@
 	name = "cyborg energy sword"
 	sword_color_icon = "red"
 	/// The cell cost of hitting something.
-	var/hitcost = 50
+	var/hitcost = 50 KILO JOULES
```

Comment:
```suggestion
	var/hitcost = 0.05 * STANDARD_CELL_CHARGE
```

Prevent future issues if someone ever decides to scale the standard cell charge for whatever reason.